DigitalOffice/TimeManagementService/DataBase/Entities/WorkTime.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace TimeManagementService.DataBase.Entities
+{
+    public class WorkTime
+    {
+        [Key]
+        public Guid Id { get; set; }
+        public Guid WorkerId { get; set; }
+        public DateTime Date { get; set; }
+        public TimeSpan Time { get; set; }
+        public string Title { get; set; }
+        public Guid Project { get; set; }
```

Comment:
This property should be called "ProjectId", as WorkerId

Reply:
fixed